utils/common/src/main/java/org/apache/brooklyn/util/javalang/Reflections.java

```diff
@@ -204,6 +209,7 @@ public Object loadInstance(String classname) throws ReflectionNotFoundException,
     }
 
     /** does not look through ancestors of outer class */
+    /** @deprecated since 0.11.0, use {@link org.apache.brooklyn.util.core.ClassLoaderUtils} in a combination with {@link #invokeConstructorFromArgs(Class, Object...)} instead */
```

Comment:
`Reflections` is in `utils-common`, while `ClassLoaderUtils` is in `core`. Could move the string based methods to another utility class into `core`.  If the caller is going to be constructing the `ClassLoaderUtils` there's not much to gain from it though.